src/client/components/Sorting/Sorting.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Sorting from './Sorting.component';
+import SortingCategories from './SortingCategories.component';
+import SortingOptions from './SortingOptions.component';
+
+export default { title: 'Some title' };
+
+export const sorting = () => <Sorting />;
+export const sortingCategories = () => <SortingCategories />;
```

Comment:
if you try to see these component in storybook they will show some errors and not run properly, and that's because these component are expecting to receive props, but they are being used here without any props.

in the case of `sortingCategories` we can do it this way
```
export const sortingCategories = () => (
  <SortingCategories category={{ name: 'category 1', id: 1 }} />
);
```

the same thing needs to be done the`Sorting` and `sortingOptions`

Reply:
On a similar note, `Sorting component` renders both `SortingCategories` and `sortingOptions` components


By only adding `Sorting` to `stroybook` then we can see both `SortingCategories` and `sortingOptions` without creating extra stories for them (also they are small and not really usefully on their own to create separate stories for them)